### ISSUE TITLE
[Bug]: User login is hung by mo  with tae-cn-tae-dn running on k8s 

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#6246]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):nightly-664f4603(docker image tag)
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

when login to mo using dump  with tae-cn-tae-dn running on k8s , login is hung by mo.
and several minute later, when relogin to mo , mo return ERROR 1045 (28000): Access denied for user

error log is attanched in Additional information

### Additional information

[cn-20221103.log](https://github.com/matrixorigin/matrixone/files/9930131/cn-20221103.log)

**Comment by daviszhen** [comment]
the log has a special error when resolving the `mo_account` table. it needs to be fixed first.

```
{"level":"INFO","time":"2022/11/03 13:13:52.059688 +0000","caller":"fileservice/local_fs.go:220","msg":"error: file 2a6efb63-3670-4110-bdd5-5fa9a55746b5 is not found","span":{}}
```

![WechatIMG1433](https://user-images.githubusercontent.com/60595215/199865405-aea5d55c-8abd-4476-8d80-ad048d3e6053.jpeg)
![WechatIMG1427](https://user-images.githubusercontent.com/60595215/199865428-488ad6f5-5e0b-40be-9f29-68728c3cd6f1.jpeg)

**Comment by XuPeng-SH** [comment]
The root cause is that both cn and dn use the local filesystem as backend storage. It should be fixed when s3 related configuration changes is done

**Comment by XuPeng-SH** [comment]
s3 works and it should be fixed.

**Comment by aressu1985** [comment]
fixed: e23e00f1d01d43e33c509a6d239ccc38b5f93409
